app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
```

Comment:
you can unpack these variables in the easiest way, also use the more descriptive names
command, old_file, new_file = command.split()